include/envoy/event/scaled_range_timer_manager.h

```diff
@@ -53,6 +66,17 @@ class ScaledTimerMinimum : private absl::variant<ScaledMinimum, AbsoluteMinimum>
     return absl::visit<Visitor, const absl::variant<ScaledMinimum, AbsoluteMinimum>&>(
         Visitor(maximum), *this);
   }
+
+  inline bool operator==(const ScaledTimerMinimum& other) const {
```

Comment:
Would this work as an alternate way to delegate to the (private) base class method?
using absl::variant<ScaledMinimum, AbsoluteMinimum>::operator==;

Reply:
No, because it's not defined as a method. I hoped that removing the operator== declaration entirely would work but the compiler doesn't find the free operator== function for absl::variant because of the private inheritance.